changelog/6197.bugfix.rst

```diff
@@ -0,0 +1 @@
+Revert "Fix fail skipping the first test in package marked as ``skip``".
```

Comment:
Note that this was edited later:

> - `#5830 <https://github.com/pytest-dev/pytest/issues/5830>`_: The first test in a package (``__init__.py``) marked with ``@pytest.mark.skip`` is now correctly skipped.

I think it makes sense to include it (stating that a feature/change from the last release was removed) with 6194.